src/js/utils/ErrorMessageUtil.js

```diff
@@ -0,0 +1,71 @@
+const MESSAGE_VARIABLE = /\{\{([^\}]+)\}\}/g;
+
+const ErrorMessageUtil = {
+
+  /**
+   * This function returns the `message` of the given array object,
+   * with the path and message components concatenated into a uniform
+   * string that can be displayed unanchored.
+   *
+   * @param {Object} error - The error message
+   * @param {Array} pathTranslationRules - The path translation rules
+   * @returns {String} Returns the composed error string
+   */
+  getUnanchoredErrorMessage(error, pathTranslationRules) {
+    const pathString = error.path.join('.');
+    const rule = pathTranslationRules.find(function (rule) {
+      return rule.match.exec(pathString);
+    });
+
+    // If no translation found, prefix with the path ID
+    if (!rule && !pathString) {
+      return error.message;
+    }
+    if (!rule) {
+      return `${pathString}: ${error.message}`;
+    }
+
+    // Otherwise prefix with the name from the rule
```

Comment:
In comments I'd like to see not `what` but rather `why`.
Maybe I'm out of the context but I don't know why do we want to prefix or lowercase something.
Maybe a small chat would help